website/docs/cloud-docs/overview/activate-flex.mdx

```diff
@@ -41,7 +41,7 @@ After you create your user accounts:
 
 Organization members may be owners or admins in only the Terraform Cloud or HCP organization. To safeguard access to your Flex balance, determine whether to invite Terraform Cloud organization owners to the HCP organization, or whether to invite HCP admins to Terraform Cloud.  
 
-**Example**: Multi-Tenant (i.e. Managed Services )
+**Example**: Multi-Tenant (i.e. Managed Services)
```

Comment:
```suggestion
### Example scenario: Invite multiple tenants to your managed services
```
There are two instances of this construction on this page, but it's not real clear what they mean. Are they functioning as headings for the following paragraphs? Or are we supposed to read these as the concluding sentences in their previous paragraphs? My suggestion assumes that the we mean the former. In either case, there are multiple style guide violations (use of bold face, use of Latin-based abbreviations, use of parentheses outside of code snippet) that we should address for both constructions.

Reply:
Good call, perhaps they would serve better as h3 headers. I'm going to address these two sections directly in a commit, so I'll mark this comment as resolved.

Reply:
I actually took a slightly different approach and listed the two examples as a bulleted list, and added some context around the examples. Let me know if you think this is a step in the right direction.